tsl/test/expected/hypertable_generalization.out

```diff
@@ -0,0 +1,495 @@
+               show_chunks               
+-----------------------------------------
+ _timescaledb_internal._hyper_5_17_chunk
+(1 row)
+
+SELECT show_chunks('jobs_big_serial', newer_than => 500);
+               show_chunks               
+-----------------------------------------
+ _timescaledb_internal._hyper_5_18_chunk
+(1 row)
+
+-- Verify drop_chunks API
+SELECT count(*) FROM timescaledb_information.chunks WHERE hypertable_name =
+'jobs_big_serial';
+ count 
+-------
+     4
+(1 row)
+
+SELECT drop_chunks('jobs_big_serial', newer_than => 500);
+               drop_chunks               
+-----------------------------------------
+ _timescaledb_internal._hyper_5_18_chunk
+(1 row)
+
+SELECT count(*) FROM timescaledb_information.chunks WHERE hypertable_name = 'jobs_big_serial';
+ count 
+-------
+     3
+(1 row)
+
+SELECT drop_chunks('jobs_big_serial', newer_than => 200, older_than => 300);
+               drop_chunks               
+-----------------------------------------
+ _timescaledb_internal._hyper_5_17_chunk
+(1 row)
+
+SELECT count(*) FROM timescaledb_information.chunks WHERE hypertable_name = 'jobs_big_serial';
+ count 
+-------
+     2
+(1 row)
+
+DROP TABLE jobs_big_serial;
+-- Verify partition function
+CREATE TABLE test_table_int(id TEXT, device INTEGER, time TIMESTAMPTZ);
+SELECT create_hypertable('test_table_int', by_range('id', 10, partition_func => 'part_func'));
+NOTICE:  adding not-null constraint to column "id"
+ create_hypertable 
+-------------------
+ (6,t)
+(1 row)
+
+INSERT INTO test_table_int VALUES('1', 1, '01-01-2023 11:00'::TIMESTAMPTZ);
+INSERT INTO test_table_int VALUES('10', 10, '01-01-2023 11:00'::TIMESTAMPTZ);
+INSERT INTO test_table_int VALUES('29', 100, '01-01-2023 11:00'::TIMESTAMPTZ);
+SELECT count(*) FROM timescaledb_information.chunks WHERE hypertable_name = 'test_table_int';
+ count 
+-------
+     3
+(1 row)
+
+DROP TABLE test_table_int;
+DROP FUNCTION part_func;
+-- Migrate data
+CREATE TABLE test_table_int(id INTEGER, device INTEGER, time TIMESTAMPTZ);
+INSERT INTO test_table_int SELECT t, t%10, '01-01-2023 11:00'::TIMESTAMPTZ FROM generate_series(1, 50, 1) t;
+SELECT create_hypertable('test_table_int', by_range('id', 10), migrate_data => true);
+NOTICE:  adding not-null constraint to column "id"
+NOTICE:  migrating data to chunks
+ create_hypertable 
+-------------------
+ (7,t)
+(1 row)
+
+SELECT count(*) FROM timescaledb_information.chunks WHERE hypertable_name = 'test_table_int';
+ count 
+-------
+     6
+(1 row)
+
+DROP TABLE test_table_int;
+-- Create default indexes
+CREATE TABLE test_table_int(id INTEGER, device INTEGER, time TIMESTAMPTZ);
+SELECT create_hypertable('test_table_int', by_range('id', 10), create_default_indexes => false);
+NOTICE:  adding not-null constraint to column "id"
+ create_hypertable 
+-------------------
+ (8,t)
+(1 row)
+
+SELECT indexname FROM pg_indexes WHERE tablename = 'test_table_int';
```

Comment:
Could you also execute this statement on the previous version of the `test_table_int` hypertable (e.g., in line 350) to show that indexes are created by default?

Reply:
Added.